.golangci.yml

```diff
@@ -83,6 +83,9 @@ linters-settings:
 linters:
   enable-all: true
   disable:
+    - godot
+    - nestif
+    - testpackage
```

Comment:
why are we disabling these?

Reply:
I have updated golangci to support t.Cleanup function (added in go 1.4). These linters were added in the new version.

Reply:
Can we add an issue and comment to followup on these news linters? I haven't looked into what they analyze.

Reply:
Sure, looks like we need `nestif` (for sure). We do not need `testpackage`(for sure). And probably we need `godot`.
godot is about to add `.` at the end of comments.

Reply:
@troyronda FYI: it is not the latest version of `golangci.` There are even more linters. I will create an issue to update the  `golangci` to the latest version.

Reply:
Sounds good. @soluchok

Reply:
I've created an issue https://github.com/hyperledger/aries-framework-go/issues/2032